src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -352,13 +397,38 @@
             if (sourceContainer.isEqualNode(targetContainer)) {
                 reorderFields();
             } else {
+                createFieldDefinitionNode(fieldRendered);
                 addField();
             }
 
-            currentActiveGroup?.classList.remove('ibexa-collapse--active-field-definitions-group');
-            targetContainerGroup.classList.add('ibexa-collapse--active-field-definitions-group');
-
+            setActiveGroup(targetContainerGroup);
             removeDragPlaceholders();
+
+            global.setTimeout(() => {
+                removeHighlight();
+            }, TIMEOUT_REMOVE_HIGHLIGHT);
+        }
+
+        getPlaceholderNode(target) {
+            const draggableItem = target.closest(`${this.selectorItem}:not(${this.selectorPlaceholder})`);
+
+            if (draggableItem) {
+                return draggableItem;
+            }
+
+            if (this.emptyContainer.contains(target)) {
+                return this.emptyContainer;
+            }
+
+            if (this.anchoredPlaceholder.contains(target)) {
+                return this.anchoredPlaceholder;
+            }
+
+            return null;
+        }
+
+        getPlaceholderPosition(item, event) {
```

Comment:
`position` I would expect `x,y` here is just `y` so maybe `PositionTop` or `PositionY`?